training/cli.py

```diff
@@ -7,19 +7,26 @@
 import requests
 
 
+class AuthentificationError(Exception):
+    def __init__(self, message=None, errors=None):
+        if errors and "UnrecognizedClientException" in str(errors):
```

Comment:
question: what would the default error message be if `errors = None` and that if statement is false